apps/speed.c

```diff
@@ -368,6 +419,9 @@ OPTIONS speed_options[] = {
 #ifndef NO_FORK
     {"multi", OPT_MULTI, 'p', "Run benchmarks in parallel"},
 #endif
+#ifndef ASYNC_NULL
+    {"async_jobs", OPT_ASYNCJOBS, 'p', "Enable async mode and start N jobs"},
```

Comment:
Help text should probably say "Enable async mode and start pnum jobs". The help text currently looks like this:
````
 -async_jobs pnum  Enable async mode and start N jobs
````